external/src/main/java/com/redhat/cloud/policies/engine/actions/plugins/NotificationActionPluginListener.java

```diff
@@ -35,16 +37,18 @@
 import java.util.Set;
 
 /**
- * WebhookActionPluginListener sends a JSON encoded message in the format understood by the
+ * NotificationActionPluginListener sends a JSON encoded message in the format understood by the
  * notifications-backend in cloud.redhat.com. The encoding schema is defined in Avro and generated during
  * the compile.
  */
-@Plugin(name = "webhook")
```

Comment:
I was worried that this might break on old data, but the engine discards the actions with missing plugin and logs it(debug):
https://github.com/RedHatInsights/policies-engine/blob/fc5190d330c93febf6aba7b19df6374f1180aebc/external/src/main/java/com/redhat/cloud/policies/engine/actions/QuarkusActionPluginListener.java#L54